presentation/src/main/java/com/knocklock/presentation/lockscreen/LockNotiItem.kt

```diff
@@ -0,0 +1,135 @@
+package com.knocklock.presentation.lockscreen
+
+import android.graphics.Bitmap
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.knocklock.presentation.ui.theme.KnockLockTheme
+
+/**
+ * @Created by 김현국 2022/12/02
+ * @Time 3:06 PM
+ */
+
+data class Notification(
+    val icon: Bitmap?,
+    val appTitle: String,
+    val notiTime: String,
+    val title: String,
+    val content: String
+) {
+    companion object {
+        val Test = Notification(
+            icon = null,
+            appTitle = "Kakao",
+            notiTime = "Now",
+            title = "KnockLock app 개발중",
+            content = "이거 너무나 재미가 있는걸~"
+        )
+    }
+}
+
+@Composable
+fun LockNotiItem(
+    modifier: Modifier = Modifier,
+    notification: Notification = Notification.Test
+) {
+    Column(
+        modifier = modifier,
+        verticalArrangement = Arrangement.SpaceBetween
+    ) {
+        Spacer(modifier = Modifier.height(4.dp))
+        LockNotiTop(
+            modifier = Modifier.padding(horizontal = 10.dp),
+            icon = null,
+            appTitle = notification.appTitle,
+            time = notification.notiTime
+        )
+        LockNotiContent(
+            modifier = Modifier.padding(horizontal = 10.dp).wrapContentHeight(),
+            title = notification.title,
+            content = notification.content
+        )
+        Spacer(modifier = Modifier.height(4.dp))
+    }
+}
+
+@Composable
+fun LockNotiTop(
+    modifier: Modifier = Modifier,
+    icon: Painter?,
```

Comment:
```suggestion
    @Drwable iconRes: Int?,
```

Drawable Res로 대신 Painter를 넘기는 이유가 있을까요?? 🦅

Reply:
@jaeryo2357 아직 타 앱의 아이콘을 접근하는 방법에 대해 알아보지 못해서 일단 Painter로 넣었는데, 외부에서 painterResource를 넣어줘도 되고, DrawableRes :id를 넣어줘도 될것같아용
`EX ) val painter: Painter = painterResource(id = R.drawable.ic_worldcup)`

Reply:
@jaeryo2357  확장성 고려를 위해서 Drawable을 넘기는방식으로 수정할께용